api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -114,6 +129,18 @@ message RBAC {
   // Maps from policy name to policy. A match occurs when at least one policy matches the request.
   // The policies are evaluated in lexicographic order of the policy name.
   map<string, Policy> policies = 2;
+
+  // Condition for the audit logging to be happen. Specific to HTTP filters.
```

Comment:
I don't actually think there's anything specific to HTTP filters about either of these fields.  I think audit logging could just as easily be supported for L4 filters.

Reply:
Makes sense. It's mainly that as L4 filters, it won't be able to log as much as the context we intend to include.

Speaking of this, I suppose the audit context (or maybe Envoy prefers to call it metadata) should be technically part of the API but not reflected in a proto, unless we have a service logger (like the [grpc access logger](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/access_loggers/grpc/v3/als.proto)). What would be the general approach to document that besides our gRFC?

Reply:
Forgot to say, I removed this sentence, "Specific to HTTP filters."

Reply:
I think the context passed to the audit logging plugin is not really a function of the xDS API, so I don't think it needs to be specified here.  I think that context is actually a function of the extension API that the data plane provides to implement that plugin.  Each data plane is likely to do that extension API a little bit differently.

Reply:
I see. This makes a lot of sense. Thank you, Mark!